process/peer/process.go

```diff
@@ -122,9 +122,26 @@ func NewValidatorStatisticsProcessor(arguments ArgValidatorStatisticsProcessor)
 
 	ratingReaderSetter.SetRatingReader(rr)
 
+	listIndexUpdaterSetter, ok := rater.(sharding.ListIndexUpdaterSetter)
+	if !ok {
+		return nil, process.ErrNilRatingReader
```

Comment:
It was my bad when creating the ratingReader and returning ErrNilRatingReader. Could you please change line 115 and 127 with 2 other errors? ErrRaterNotRatingReaderSetter ErrRaterNotListIndexUpdaterSetter

Reply:
done